src/main/scala/com/codacy/duplication/pmd/ScalaTokenizer.scala

```diff
@@ -15,13 +15,13 @@ private[pmd] object ScalaTokenizer extends Tokenizer {
 
     val fileName = sourceCode.getFileName
 
-    Try(new File(fileName).parse[Source]) match {
-      case Success(Parsed.Success(tree)) =>
+    Try(new File(fileName).parse[Source].toEither) match {
```

Comment:
Here I was getting a strange compile error like "The following match could fail on `Success(Success())`"
Converting to `Either` fixes the problem